misc/checktor.js

```diff
@@ -0,0 +1,33 @@
+/*
+  Copyright 2015 - Hermes Center - GlobaLeaks project
+  Author <giovanni.pellerano@evilaliv3.org>
+
+  Javascript CheckTor library
+*/
+
+function redirectIfOnTor(url, test_url) {
+  // Test if the user is using Tor and in that case
+  // redirects the user to provided url
+  try {
+    if (typeof(test_url) === 'undefined') {
+      var test_url = 'https://antani.tor2web.org/checktor';
+    }
+    if (window.XMLHttpRequest) {
+      var xmlhttp = new XMLHttpRequest();
+
+      xmlhttp.onreadystatechange=function() {
+        if (xmlhttp.readyState==4 && xmlhttp.status==200) {
```

Comment:
Codacy has a fix for the issue: [Replace `==4·&&·xmlhttp.status==` with `·==·4·&&·xmlhttp.status·==·`](https://app.codacy.com/gh/tor2web/Tor2web/pullRequest?prid=9785887)
```suggestion
        if (xmlhttp.readyState == 4 && xmlhttp.status == 200) {
```